authors/apps/authentication/serializers.py

```diff
@@ -179,9 +179,14 @@ class UserSerializer(serializers.ModelSerializer):
         write_only=True
     )
 
+    profile = ProfileSerializer(write_only=True)
+
+    bio = serializers.CharField(source='profile.bio', read_only=True)
+    image = serializers.CharField(source='profile.image', read_only=True)
+
     class Meta:
         model = User
-        fields = ('email', 'username', 'password', 'token',)
```

Comment:
why did you remove this?

Reply:
Same thing here, I didn't delete the entire line but rather deleted the token field and modified it to add other user profile fields. these fields are later displayed as user's information/profile.